src/router_core/agent_address.c

```diff
@@ -54,6 +56,8 @@ const char *qdr_address_columns[] =
      "deliveriesTransit",
      "deliveriesToContainer",
      "deliveriesFromContainer",
+     "deliveriesToRouteContainer",
+     "deliveriesFromRouteContainer",
```

Comment:
I find the names of these two new attributes misleading.  They appear to be grouped with the former two attributes "deliveriesToContainer" and "deliveriesFromContainer" because of the similar naming.  The new stats are supposed to be grouped with "deliveriesIngress" and "deliveriesEgress".  I think better names would be "deliveriesIngressRouteContainer" and "deliveriesEgressRouteContainer".